csaf_2.1/json_schema/csaf_json_schema.json

```diff
@@ -1288,6 +1288,36 @@
                     "cvss_v4": {
                       "$ref": "https://www.first.org/cvss/cvss-v4.0.json"
                     },
+                    "epss": {
+                      "title": "EPSS",
+                      "description": "Contains the EPSS data.",
+                      "type": "object",
+                      "required": [
+                        "percentile",
+                        "probability",
+                        "timestamp"
+                      ],
+                      "properties": {
+                        "percentile": {
+                          "title": "Percentile",
+                          "description": "Contains the rank ordering of probabilities from highest to lowest.",
+                          "type": "string",
+                          "pattern": "^(([0]\\.([0-9])+)|([1]\\.[0]+))$"
+                        },
+                        "probability": {
+                          "title": "Probability",
+                          "description": "Contains the likelihood that any exploitation activity for this Vulnerability is being observed in the 30 days following the given timestamp.",
+                          "type": "string",
+                          "pattern": "^(([0]\\.([0-9])+)|([1]\\.[0]+))$"
```

Comment:
I wonder if we can declare patterns upfront and reuse them.

Reply:
I though about that last night as well. My conclusion was: For only 2 instances with just patterns, the overhead is too big.